dash-pipeline/bmv2/stages/ha.p4

```diff
@@ -0,0 +1,133 @@
+#ifndef _DASH_STAGE_HA_P4_
+#define _DASH_STAGE_HA_P4_
+
+control ha_stage(inout headers_t hdr,
```

Comment:
@r12f, as discussed can we add a capability attribute for DPU scope versus ENI scope HA.

Reply:
As chatted, I believe it will be better to add it in another PR.

The DPU HA capability will make more sense to be added along with the DPU API or a dedicated one, if needed. Putting the DPU HA capability into ENI HA API will be really weird...

Reply:
Actually, let me get some capacities in there, we can probably model this in a generic way.